src/topology/instances/real.lean

```diff
@@ -340,4 +325,27 @@ begin
   exact bounded.subset I bounded_closed_ball
 end⟩
 
+lemma real.image_Icc {f : ℝ → ℝ} {a b : ℝ} (hab : a ≤ b) (h : continuous_on f $ Icc a b) :
+f '' Icc a b = Icc (Inf $ f '' Icc a b) (Sup $ f '' Icc a b) :=
```

Comment:
Could you please prove that a compact (or closed bounded?) connected subset of a conditionally complete linear ordered topological space is `Icc (Inf s) (Sup s)`? Then the theorem will work, e.g., for `nnreal` and `ennreal`.

Reply:
Note that we already have `is_connected.Ioo_cInf_cSup_subset`.

Reply:
I'm going to prove&push this while you're asleep.

Reply:
Done. Feel free to (partially) revert.

Reply:
Nice! I'll let you update it to the new names, but otherwise it's all good.

bors d+